### ISSUE TITLE
Cookie g_state is invalid

### ISSUE THREAD

**Issue by hyzyla** [google/google-api-javascript-client#844]
**Summary**
I'm trying to integrate the Google One Tap button into my Python web application and found out that the cookie `g_state`, which [client script](https://accounts.google.com/gsi/client ) [1] uses to manage state, has not valid value format for cookies: `g_state: {"i_l":0}`. Because of that, the parser (std) on the Python side fails to parse such cookie value. 

```
Python 3.9.15 (main, Oct 25 2022, 06:48:03) 

In [1]: from http.cookies import SimpleCookie

In [2]: SimpleCookie('a=b; g_state={"i_l":0}; c=d;')
Out[2]: <SimpleCookie: a='b'> 
```

I'm also not the first person who encountered with the same problem: 
1. [question](https://stackoverflow.com/questions/66864766/one-tap-login-g-state-cookie-is-not-parseable-on-http-cookies-simplecookie) on StackOverflow
2. [bug report ](https://bugs.python.org/issue25228)on the Python issue tracker
3. [discussion](https://groups.google.com/a/chromium.org/g/chromium-discuss/c/T3oHExJMr0M) on Chromium-discuss.

Could you change value of the cookie to something else, which will be more compatible with Cookie specs?

**Comment by dvanderb** [comment]
This looks like a question about Google Login and Authentication. Please address this question to the google-signin tag on stackoverflow(https://stackoverflow.com/questions/tagged/google-signin)

If you believe this is in error, please reopen the issue.

**Comment by hyzyla** [comment]
Hi, @dvanderb, I believe this is a bug, not question for StackOverflow how to integrate library or so. So I'll reopen this  issue again if you don't mind. 

Issue that this library sets invalid by spec cookie value that breaks systems that follows spec

**Comment by dvanderb** [comment]
> Hi, [@dvanderb](https://github.com/dvanderb), I believe this is a bug, not question for StackOverflow how to integrate library or so. So I'll reopen this issue again if you don't mind.
> 
> Issue that this library sets invalid by spec cookie value that breaks systems that follows spec

Yes I understand.. but this part of the library is supported on Stack overflow .
